operators/index.js

```diff
@@ -0,0 +1,145 @@
+function add(){ 
+
+   let num1=document.getElementById("num1").value; 
```

Comment:
What will happen, if user enters string in the text box? How you will resolve it?